aspnetcore/migration/rc1-to-rtm.md

```diff
@@ -1,466 +0,0 @@
----
-title: Migrating from ASP.NET 5 RC1 to ASP.NET Core 1.0 | Microsoft Docs
```

Comment:
Instead of deleting the files, it would be best to leave redirect metadata. Best redirect URL would probably be https://docs.microsoft.com/aspnet/core/migration/index